xml/deployment_prep_x86_nsysreqs.xml

```diff
@@ -72,119 +72,55 @@
    <term>Hard Disk Requirements</term>
    <listitem>
     <para>
-     The disk requirements depend largely on the installation selected and how
-     you use your machine.  Commonly, you need more space than the installation
-     software itself needs to have a system that works properly. Minimum
-     requirements for different selections are:
+     The hard disk requirements do not only depend on the selected product and
+     the packages installed, but also on how you use your machine. In addition
+     to the space occupied by the software itself, you need some more space for
+     a system to work properly.
     </para>
-    <informaltable>
+    <para>
+     Especially if you want to enable Btrfs snapshots on the root volume to
+     enable system rollbacks (see <xref linkend="cha.snapper"/>), the amount of
+     hard disk space is crucial. The more snapshots you want to store and older
```

Comment:
Seems like this sentence needs an extra "the". Suggestion: "The more snapshots you want to store and *the* older..."

Reply:
Argh, good catch!